stdlib/LinearAlgebra/src/generic.jl

```diff
@@ -1897,3 +1897,48 @@ end
 
 normalize(x) = x / norm(x)
 normalize(x, p::Real) = x / norm(x, p)
+
+"""
+    copytotri!(B, A, uplo) -> B
+
+Copies a triangular part of a matrix `A` to another matrix `B`.
+`uplo` specifies the part of the matrix `A` to be copied to `B`.
+Set `uplo = 'L'` for the lower triangular part or `uplo = 'U'
+for the upper triangular part.
+
+!!! compat "Julia 1.11"
+    `copytotri!` requires at least Julia 1.11.
+
+# Examples
+```jldoctest
+julia> A = [1 2 ; 3 4];
+
+julia> B = [0 0 ; 0 0];
+
+julia> copytotri!(B, A, 'L')
+2×2 Matrix{Int64}:
+ 1  0
+ 3  4
+```
+"""
+function copytotri!(B::AbstractMatrix, A::AbstractMatrix, uplo::AbstractChar)
```

Comment:
It appears that `LinearAlgebra` already has an internal function `copytri!` that does something somewhat different: symmetrize a matrix by copying the upper/lower triangular part onto the other section. I wonder if we should avoid the similarity in names here, as it doesn't parallel the `copy-copyto!` semantics. Perhaps `copytri!` should really be renamed to indicate that it symmetrizes the matrix?